tests/test_collections.py

```diff
@@ -47,7 +47,7 @@ def setUpClass(cls):
         cls.contents = [cls.col_file_item] * 8 + [cls.col_ptr_item] * 8
         cls.uuid, cls.version = cls._put(cls, cls.contents)
         cls.invalid_ptr = dict(type="foo", uuid=cls.file_uuid, version=cls.file_version, fragment="/xyz")
-        cls.replicas = ('aws', 'gcp')
+        cls.replicas = ('aws', )  # only necessary to test in AWS
```

Comment:
Is this variable only used for paging tests? If so, can we please change the variable name to `cls.paging_test_replicas`.

Reply:
Done.